conf.d/splunk_topology.yaml.example

```diff
@@ -0,0 +1,31 @@
+# Splunk topology intake uses basic authentication
+
+init_config:
+  default_timeout: 10
+  max_retry_count: 5
+  seconds_between_retries: 1
+
+  # Polling interval between calls to splunk
+  polling_interval: 15
+
+instances:
+  - url: "http://localhost:13001"
+    username: "admin"
+    password: "admin"
+    polling_interval: 15
+    saved_searches:
```

Comment:
Maybe its an idea to have a component_searches and relation_searches list. So the user doesn't have to configure the element_type on each search. Just an idea... let me know what you think about it.